.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: CI
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
```

Comment:
3.11 should be available now and 3.6 is EOL

Reply:
Right, I'm just going off of what was already there in the `tox.ini`. Think we should drop support for <3.8 (matches the ebuild's PYTHON_TARGETS) and add 3.11 (in preparation)?